src/config.rs

```diff
@@ -49,6 +49,24 @@ pub struct PluginConfiguration {
     #[serde(with = "serde_regex")]
     pub exclude_urls: Vec<Regex>,
 
+    // Header forwarding settings
+    /// Forward the access token in a header
+    pub forward_access_token: bool,
+    /// The header name that will be used for the access token
+    /// If the header name is empty, the access token will be forwarded in the authorization header
+    pub access_token_header_name: String,
+    /// Prefix for the access token header
+    /// If the prefix is empty, the access token will be forwarded without a prefix
+    pub access_token_header_prefix: String,
```

Comment:
I think it would be preferable to use an `Option` for this so the value can be omitted from the config if not needed.